packages/wrangler/src/dev-registry.ts

```diff
@@ -48,7 +48,7 @@ async function isPortAvailable() {
 				netServer.close();
 				resolve(true);
 			});
-		netServer.listen(DEV_REGISTRY_PORT);
+		netServer.listen(DEV_REGISTRY_PORT, "127.0.0.1");
```

Comment:
This seems to work just fine:
```suggestion
		netServer.listen(DEV_REGISTRY_PORT, "localhost");
```

Can we change 127.0.0.1 everywhere to `localhost` for IPv6(-only) support?